src/components/metadata/styles/metadata.scss

```diff
@@ -0,0 +1,234 @@
+  flex-grow: 0;
+  flex-shrink: 0;
+  margin: 0.25em -8px 0 0.8em;
+
+  &,
+  & svg {
+    width: 24px;
+    height: 24px;
+  }
+}
+
+.pipeline-metadata__header,
+.pipeline-metadata__label,
+.pipeline-metadata__row {
+  margin: 1.9em 36px;
+  font-size: 1.48em;
+}
+
+.pipeline-metadata__icon {
+  display: inline-block;
+  flex-grow: 0;
+  flex-shrink: 0;
+  width: 30px;
+  height: 30px;
+  margin: 0 0.85em 0 -0.3em;
+
+  .kui-theme--light & {
+    fill: $color-text-light;
+  }
+
+  .kui-theme--dark & {
+    fill: $color-text-dark;
+  }
+}
+
+.pipeline-metadata__title {
+  flex-grow: 1;
+  margin: 0;
+  font-weight: normal;
+  font-size: 1.38em;
+  line-height: 1.6;
+  word-break: break-word;
+
+  .kui-theme--light & {
+    color: rgba($color-text-light, 0.8);
+  }
+
+  .kui-theme--dark & {
+    color: rgba($color-text-dark, 0.8);
+  }
+}
+
+.pipeline-metadata__list {
+  margin: 0;
+}
+
+.pipeline-metadata__label {
+  margin-bottom: 0.7em;
+  font-weight: normal;
+  user-select: none;
+
+  .kui-theme--light & {
+    color: rgba($color-text-light, 0.55);
+  }
+
+  .kui-theme--dark & {
+    color: rgba($color-text-dark, 0.55);
+  }
+}
+
+.pipeline-metadata__row {
+  margin-top: 0.7em;
+
+  .kui-theme--light & {
+    color: rgba($color-text-light, 0.55);
+  }
+
+  .kui-theme--dark & {
+    color: rgba($color-text-dark, 0.55);
+  }
+}
+
+.pipeline-metadata__value {
+  display: inline;
+  flex-grow: 0;
+  margin-right: auto;
+  line-height: 1.5;
+  word-break: break-word;
+
+  .kui-theme--light & {
+    color: rgba($color-text-light, 0.8);
+  }
+
+  .kui-theme--dark & {
+    color: rgba($color-text-dark, 0.8);
+  }
+}
+
+.pipeline-metadata__value--kind-token {
+  display: inline-block;
+  padding: 0.18em 0.6em 0.2em 0.6em;
+  font-weight: 600;
+
+  code & {
+    padding: 0.4em 0.8em;
+    font-weight: normal;
+    font-size: 0.9em;
+    font-family: Consolas, Monaco, 'Courier New', Courier, monospace;
+  }
+
+  .kui-theme--light & {
+    background: $color-bg-light-2;
+  }
+
+  .kui-theme--dark & {
+    background: $color-bg-dark-5;
+  }
+}
+
+$list-inline-spacing: 0.2em;
+
+.pipeline-metadata__value-list--inline {
+  display: flex;
+  flex-wrap: wrap;
+  margin: 0 -$list-inline-spacing * 1.5;
+  padding: 0;
+  list-style: none;
+}
+
+.pipeline-metadata__value-list--inline li {
+  display: inline;
+  margin: $list-inline-spacing $list-inline-spacing * 1.5;
+}
+
+.pipeline-metadata__value-list--commas li:not(:last-child):after {
+  content: ', ';
+}
+
+.pipeline-metadata__toolbox-container {
+  position: relative;
+  display: flex;
+}
+
+.pipeline-metadata__toolbox {
+  display: block;
+  margin: -0.2em 1em;
+  padding: 0;
+  list-style: none;
+}
+
+.pipeline-metadata__run-command-value {
+  display: flex;
+  flex-grow: 1;
+  align-items: center;
+}
+
+.pipeline-metadata__run-command-value--no-visible {
+  opacity: 0;
+}
+
+.pipeline-metadata__toolbox .pipeline-metadata__copy-button {
+  width: 1.65em;
+  height: 1.65em;
+  opacity: 0.55;
+}
```

Comment:
This should probably be outside of the `<code>` element so that it doesn't inherit the font-family

Reply:
That was intentional as I found the font changing this way to be jarring, I've made the change though if you want to try it out.

Reply:
Thanks, yeah I definitely find it better with the changed font, and I think Gabriel was on board with that too. Cheers!